modules/data/pending_post.py

```diff
@@ -140,25 +140,37 @@ def get_votes(self, vote: bool) -> int:
                                     where="g_message_id = %s and group_id = %s and is_upvote = %s",
                                     where_args=(self.g_message_id, self.group_id, vote))
 
-    def get_list_admin_votes(self, vote: bool) -> Optional[list[str]]:
+    def get_list_admin_votes(self, vote: 'bool | None' = None) -> 'Optional[list[str]] | Optional[list[str, bool]]':
```

Comment:
this should work as well
```suggestion
    def get_list_admin_votes(self, vote: 'bool | None' = None) -> 'Optional[list[str] | list[str, bool]]':
```